### PR TITLE
Allow to build research tools in container.

### PR DESCRIPTION
**Pull request by boehmseb**

### CODE REVIEW

**Comment by boehmseb** [comment]
History looks a bit wonky because I messed up pulling before merging vara-dev again and then I had to rebase...

File: docs/source/tutorials/container_guide.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+       vara-container select -t vara
+
+   Now you can create the base images:
+
+   .. code-block:: console
+
+       vara-container build
+
+   You can use the flag ``-i <base_image>`` to only build a specific base image.
```
Comment (vulder) [inline docs/source/tutorials/container_guide.rst]:
```suggestion
   You can use the flag ``-i <base_image>`` to only build a specific base image and should add ``--export`` if you want to use the image with slurm.
```

File: varats/varats/tools/driver_build_setup.py [hunk 2]
```diff
@@ -24,24 +28,27 @@
...
-
-    def main(self) -> None:
-        """Start VaRA setup GUI."""
-        sys.exit(self.app.exec_())
+from varats.utils.settings import save_config, vara_cfg, bb_cfg
+
+# from PyQt5.QtCore import Qt
+# from PyQt5.QtWidgets import QApplication
+
+# TODO: remove or move to other module
```
Comment (vulder) [inline varats/varats/tools/driver_build_setup.py]: What's the plan here?

**Review by vulder — APPROVED** [review]

File: varats/varats/tools/driver_build_setup.py (hunk 2 above)
Comment (boehmseb) [reply varats/varats/tools/driver_build_setup.py]: There is no plan yet. You mentioned that you want to keep the GUI functionality, so we will have to move this code somewhere else.

File: tests/tools/test_driver_buildsetup.py [hunk 3]
```diff
@@ -22,3 +27,31 @@ def test_init_config(self) -> None:
...
+        _build_in_container(research_tool, image_base, build_type)
+
+        mock_create.assert_called_with(image_base, research_tool)
+        mock_run.assert_called_with(
+            "localhost/debian:10_varats_vara_DEV", "build_VaRA", None, [
+                "build",
+                "vara",
+                "--build-type=DEV",
+                "--source-location=/varats_root/tools_src",
+                "--install-prefix=/varats_root/tools/VaRA_DEBIAN_10",
```
Comment (boehmseb) [inline tests/tools/test_driver_buildsetup.py]:
```suggestion
                "--install-prefix=/varats_root/tools",
```

**Pull request merged**